doc/style-guide.md

```diff
@@ -91,6 +92,103 @@ quaility.
 
 **[top](#table-of-contents)**
 
+## Object and Array Literals
+
+  - Object literals
+    - Use curly braces `{}` instead of `new Object()`.
+    - The opening curly brace (i.e., `{`) must be on the same line as the declaration, expression,
```

Comment:
If there's a graceful way to do so, it'd be good to call out that there's an exception to this rule (possibly in the exception below).